sympy/sets/sets.py

```diff
@@ -1628,6 +1628,10 @@ class FiniteSet(Set, EvalfMixin):
     >>> 3 in FiniteSet(1, 2, 3, 4)
     True
 
+    >>> members = [1, 2, 3, 4]
+    >>> FiniteSet(*members)
```

Comment:
Sure, the point is not to teach Python but have it as an example so that it's easy to see how to create a ``FiniteSet`` when the set members are created at runtime.